_pages/en_US/installing-boot9strap-(fredtool).txt

```diff
@@ -27,17 +26,32 @@ As an additional safety measure, we will perform an additional check for custom
 1. Hold the (Select) button
 1. Power on your device while still holding the (Select) button
 1. If the check was successful, you will boot to the HOME Menu and you may proceed with this guide
+1. Power off your device
 
 If you see a configuration menu, you already have CFW, and continuing with these instructions may BRICK your device! Follow [Checking for CFW](checking-for-cfw) to upgrade your existing CFW.
 {: .notice--danger}
+    
+#### Section II - BannerBomb3
+
+In this section, you will trigger the BannerBomb3 exploit using the DSiWare Management menu, which can be used to overwrite your Wi-Fi slots with hacked data. Your Wi-Fi connection settings will be temporarily overwitten while the exploit is active.
```

Comment:
![image](https://user-images.githubusercontent.com/27717444/195250218-622e582b-76df-4e25-b87f-c94b9b13159f.png)

Reply:
how it feels to chew 5 gum